artifacthub/artifacthub-repo.yml

```diff
@@ -0,0 +1,6 @@
+repositoryID: 2b7422af-5f40-489d-9f89-dfb358f9a485
+owners:
+  - name: nilekh
+    email: nilekhc@gmail.com
+  - name: sozercan
+    email: sozercan@gmail.com
```

Comment:
@ritazh @maxsmythe do you want to add your email here?

Reply:
You need an account on AH for `name`.

Reply:
@sozercan 
```
Please make sure the email used in the metadata file matches with the one you use in Artifact Hub.
```

Let me know if your email is correct.

Reply:
Looks like you have the correct email ID @sozercan. Thanks for confirming.